src/NLog/Logger-generated.tt

```diff
@@ -189,7 +189,7 @@ namespace NLog
         {
             if (Is<#=level#>Enabled)
             {
-                WriteToTargets(LogLevel.<#=level#>, message, exception);
+                WriteToTargets(LogLevel.<#=level#>, exception, message, null);
```

Comment:
why is this change needed?

Reply:
Because it referenced an obsolete internal method, that I have also removed (Leftovers from the reordering of the Exception-parameter to the front).